### PR TITLE
Issue #SB-18773 fix: Footer fields based on tenant slug value.

### PR DESCRIPTION
**Pull request by rajeshkumaravel**
- Issue #SB-18773 fix: Footer fields based on tenant slug value.

### CODE REVIEW

File: src/app/client/src/app/modules/core/components/main-footer/main-footer.component.ts [hunk 1]
```diff
@@ -28,21 +29,29 @@ export class MainFooterComponent implements OnInit, AfterViewInit {
...
+    public configService: ConfigService, private renderer: Renderer2, private cdr: ChangeDetectorRef, public userService: UserService,
+      private tenantService: TenantService
     ) {
     this.resourceService = resourceService;
   }
 
   ngOnInit() {
     this.instance = _.upperCase(this.resourceService.instance);
+    this.tenantFooter = {
+      helpCenterLink: undefined,
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/core/components/main-footer/main-footer.component.ts:44]: make this null

File: src/app/client/src/app/modules/core/components/main-footer/main-footer.component.ts [hunk 2]
```diff
@@ -67,26 +76,31 @@ footerAlign() {
...
-            const source = params.source || 'search';
-            applink = `${applink}&referrer=utm_source=${utm_source}&utm_medium=${source}&utm_campaign=dial&utm_term=${params.dialCode}`;
-          } else {
-            applink = `${applink}&referrer=utm_source=${utm_source}&utm_medium=get&utm_campaign=redirection`;
-          }
-          this.redirect(applink.replace(/\s+/g, ''));
-        });
+    const playstoreLink = _.get(this.tenantFooter, 'playstoreLink');
+    if (playstoreLink) {
+      this.redirect(playstoreLink);
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/core/components/main-footer/main-footer.component.ts:81]: coment for the igot player service URL so no UTM is needed

File: src/app/client/src/app/modules/core/components/main-footer/main-footer.component.ts (hunk 1 above)
Comment (rajeshkumaravel) [reply src/app/client/src/app/modules/core/components/main-footer/main-footer.component.ts]: Done

File: src/app/client/src/app/modules/core/components/main-footer/main-footer.component.ts (hunk 2 above)
Comment (rajeshkumaravel) [reply src/app/client/src/app/modules/core/components/main-footer/main-footer.component.ts:82]: Done

**Comment by rajeshkumaravel** [comment]
- @rajeevsathish Closing this request; as this has to be merged to another branch.

**Pull request closed without merging**